CONTRIBUTING.md

```diff
@@ -11,21 +11,40 @@ We actively welcome your pull requests.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
 ### Forking, cloning and running Rapid
-
 - Make sure to use the latest `node` with something like  `nvm install node`
 - Run `npm run all`
 - Run `npm run quickstart` to start the server
 
 ### Debugging and testing
-
 - In VSCode go to Run>Add Configuration... and select Launch Chrome
 - Examples of breakpoints
 - Conditional breakpoints
 - Some interesting files to break (init, etc)
 
+#### Code walkthrough
+
+Top-level Rapid Directories: 
+`css`: pretty self-explanatory, just contains our .css for styling things like the sidebars, top bars, buttons, etc. Side note: there is no .css 'inside' the map istelf- that's all styled using webGL. 
```

Comment:
### CSS

The 'CSS' directory is quite straightforward; it houses our .css files used for styling elements such as sidebars, top bars, buttons, and more. It's worth noting that there is no .css used 'inside' the map itself; all styling for the map is achieved through webGL.

Reply:
Thank you for these great corrections Rita. However, you don't have to write the whole document yourself. These are all corrections I was going to do on my own. Sorry for not being more clear about this before. I only copy-pasted Ben's Discord comments as placeholder text while I got to work on the writing.

That said these are all excellent reviews and I will be using them definitely, thank you!